app/controllers/api_controller.rb

```diff
@@ -43,36 +44,40 @@ def area
       "Recent applications in the area (#{lat0},#{lng0}) (#{lat1},#{lng1})")
   end
 
+  def date_scraped
+    begin
+      date = Date.parse(params[:date_scraped])
+      api_render(Application.where(date_scraped: date.beginning_of_day...date.end_of_day), "All applications collected on #{date}")
+    rescue ArgumentError => e
```

Comment:
You're right - I will change this. I just did this because of a problem when I put it around just `Date.parse` at first.